### DISCUSSION TITLE
Does the version here need to be changed to 2.5? (Advanced - Developer Debug Mode)

### DISCUSSION THREAD

**Discussion by Woukim** [lllyasviel/Fooocus#856] [Q&A] [answered]
![image](https://github.com/lllyasviel/Fooocus/assets/138323639/3a323df5-32f5-4c37-ab12-06b13718d419)

**Comment by aolko** [comment]
Should probably be 2.5 by default

**Comment by lllyasviel** [comment, marked as answer]
we are still testing 2.5
if no problem is discovered it will be 2.5 soon
otherwise we will have a model like 2.6 and continue the test
